### DISCUSSION TITLE
Adding New features

### DISCUSSION THREAD

**Discussion by samyak112** [MbrllaHacks/Website#68] [Q&A]
Hello everyone, I am new to open source and I just made my first contribution on this project, I'll be more than happy if you have any new features in mind that I can contribute to add, I can do both backend and front end. Thank you.

**Comment by FengchiW** [comment]
Dark mode?

**Reply by samyak112** [reply]
Yes I can work on that using Redux but I am not really good with selecting colours so if i can get a mockup in figma or something It'll be great.
